libs/ui/src/screens/wallet/components/assets-widget/assets-widget.styles.ts

```diff
@@ -4,7 +4,7 @@ import { getCustomSize } from '../../../../styles/format-size';
 
 export const styles = StyleSheet.create({
   widgetStyles: {
```

Comment:
rename please `widgetStyles` to `root` and `root` to `widgetContainer` or just 'container'